corehq/apps/smsbillables/migrations/0007_smsbillable_multipart_count.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsbillables', '0006_remove_smsbillable_api_response'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
I have some concerns about performance when running on prod (table has ~3 million rows).  It might be better to temporarily add the column as nullable, run a data migration (outside of deploy), and make the column non-nullable.